cpp/src/copying/concatenate.cu

```diff
@@ -360,6 +369,15 @@ std::unique_ptr<table> concatenate(std::vector<table_view> const& tables_to_conc
                            }),
                "Mismatch in table columns to concatenate.");
 
+  if (tables_to_concat.front().num_columns() > 0 &&
+      is_fixed_point(tables_to_concat.front().column(0).type())) {
+    CUDF_EXPECTS(std::all_of(tables_to_concat.cbegin(),
+                             tables_to_concat.cend(),
+                             [scale = tables_to_concat.front().column(0).type().scale()](
+                               auto const& t) { return scale == t.column(0).type().scale(); }),
```

Comment:
This should call a function `have_same_scales(first_table, t);` or check the scales in 
`have_same_types(first_table, t);`

Reply:
This was removed :)